Gemfile

```diff
@@ -9,7 +9,7 @@ group :test do
   gem 'rake'
   gem 'puppet', ENV.fetch('PUPPET_VERSION',  '~> 4.0')
   gem 'rspec'
-  gem 'rspec-puppet'
+  gem 'rspec-puppet', :git => 'https://github.com/heliocentric/rspec-puppet.git', :branch => 'fix_function_test'
```

Comment:
I forgot this has been merged upstream, I will remove my fork when I decide to go back on the clock ;)